enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/ExpressionAcceptanceTest.scala

```diff
@@ -97,15 +97,15 @@ class ExpressionAcceptanceTest extends ExecutionEngineFunSuite with CypherCompar
   test("prepending item to a list should behave correctly in all runtimes") {
     val query = "CYPHER WITH {a:[1,2,3]} AS x RETURN 'a:' + x.a AS r"
 
-    val result = succeedWith(Configs.All - Configs.EnterpriseInterpreted, query)
+    val result = succeedWith(Configs.AllExceptSleipnir, query)
```

Comment:
For the same reason we did not change Sleipnir to Enterprise-Interpreted. We are using this as a reminder, a TODO. At the time the goal was to get 100% coverage with sleipnir. That is unlikely in the short term, so perhaps we should rename this. I also see a few place where we have RegisterAllocation, and that could also be renamed, but would prefer to do that in another PR. This PR focuses on the surface and the definition of the runtime.